beacon-chain/blockchain/service.go

```diff
@@ -299,6 +405,46 @@ func (c *ChainService) processBlock(block *types.Block) error {
 	return nil
 }
 
+func (c *ChainService) processBlockNew(block *types.Block) error {
```

Comment:
are you calling `processBlockNew` anywhere?

Reply:
Please make this the canonical method and rename the deprecated one